app/lib/dns.server.ts

```diff
@@ -161,13 +161,13 @@ const isNameValid = (name: string) => {
   return /^[a-z0-9-]+.[a-z0-9-]+.[a-z]+.[a-z]+\.?$/.test(name);
 };
 
-const isValueValid = (type: RecordType, value: string) => {
+export const isValueValid = (type: RecordType, value: string) => {
   if (type === 'A') {
-    return isIPv4(value);
+    return validator.isIP(value, 4);
```

Comment:
Can you not `import { isIP } from 'validator'`, since that's all you're using in here?